scipy/stats/tests/test_stats.py

```diff
@@ -3314,11 +3327,23 @@ def test_ttest_rel():
     assert_array_almost_equal(np.abs(p), pr)
     assert_equal(t.shape, (2, 3))
 
-    t,p = stats.ttest_rel(np.rollaxis(rvs1_3D,2), np.rollaxis(rvs2_3D,2), axis=2)
+    t, p = stats.ttest_rel(np.rollaxis(rvs1_3D, 2), np.rollaxis(rvs2_3D, 2),
+                          axis=2)
```

Comment:
Indent one more space to align with the first argument above:
```suggestion
                           axis=2)
```